src/sentry/api/endpoints/project_ownership.py

```diff
@@ -161,7 +171,15 @@ def get(self, request: Request, project) -> Response:
 
         :auth: required
         """
-        return Response(serialize(self.get_ownership(project), request.user))
+        should_return_schema = features.has(
+            "organizations:streamline-targeting-context",
+            project.organization,
+        )
```

Comment:
can we add the actor here like this one https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/incident.py#L31